### PR TITLE
121 gui reflect incoming data and start to visuzalize

### PR DESCRIPTION
**Pull request by CaseyBramlett**
lots of visualizations on dashboard page

### CODE REVIEW

File: src/app/client/src/components/NetworkTableCard.tsx [hunk 1]
```diff
@@ -8,73 +9,121 @@ import {
...
-  CardContent,
-  CardDescription,
-} from "@/components/ui/card";
+import { Card, CardHeader, CardTitle, CardContent } from "@/components/ui/card";
+import { Switch } from "@/components/ui/switch";
+import { Label } from "@/components/ui/label";
 import type { FeederMessage } from "../../../types/client-types";
+import PcapMetaDataModal from "./PcapMetaDataModal";
+
+const classMap = {
```
Comment (IsaiahHarvi) [inline src/app/client/src/components/NetworkTableCard.tsx:18]: there is another model with less classes that has a different class map.  we need a good way to track which model is on the feeder for display on the gui

File: src/app/client/src/components/PieChart.tsx [hunk 2]
```diff
@@ -13,32 +13,41 @@ import {
...
-  data: string[];
-}
-
-// PieChartComponent implementation (subcomponent)
-const PieChartComponent = ({ data }: PieChartComponentProps) => {
-  // Process the array of strings and count the occurrences
-  const processedData = useMemo(() => {
-    const counts: { [key: string]: number } = {};
+// Declare the classMap with prediction values and colors
+const classMap = {
```
Comment (IsaiahHarvi) [inline src/app/client/src/components/PieChart.tsx:19]: does this have to be defined in two places?

File: src/app/client/src/middleware/api/functions/startFeeders.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { restPost } from "../../rest-calls/rest-post"; // Assuming you have a `restPost` utility for making POST requests
```
Comment (IsaiahHarvi) [inline src/app/client/src/middleware/api/functions/startFeeders.ts:1]: lol lets keep gpt comments out of main
```suggestion
import { restPost } from "../../rest-calls/rest-post";
```

File: src/app/client/src/middleware/api/functions/stopFeeders.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { restPost } from "../../rest-calls/rest-post"; // Assuming you have a `restPost` utility for making POST requests
```
Comment (IsaiahHarvi) [inline src/app/client/src/middleware/api/functions/stopFeeders.ts:1]:
```suggestion
import { restPost } from "../../rest-calls/rest-post";
```

File: src/app/client/src/components/PieChart.tsx (hunk 2 above)
Comment (CaseyBramlett) [reply src/app/client/src/components/PieChart.tsx:19]: Will fix tomorrow

**Review by IsaiahHarvi — APPROVED** [review]

**Pull request merged**